.github/workflows/build.yml

```
@@ -352,3 +369,40 @@ jobs:
       with:
         name: ${{ matrix.artifacts_name }}
         path: ${{ matrix.artifacts_path }}
+
+  update_manifest:
+    name: "Update manifest file on download server"
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+    - name: "Check out repository"
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: "Collect Artifacts Metadata & Write Manifest"
+      if: github.event_name == 'push' && env.SSH_PASSWORD != null
+      run: python3 tools/collect_artifacts_metadata.py
+      env:
+        JOB_DATA: ${{ toJSON(needs.build) }}
+        MANIFEST_URL: "https://downloads.mixxx.org/builds/{git_branch}/manifest.json"
+        DEPLOY_DESTPATH: "builds/{git_branch}"
+        SSH_PASSWORD: ${{ secrets.DOWNLOADS_HOSTGATOR_DOT_MIXXX_DOT_ORG_KEY_PASSWORD }}
+
+    - name: "Deploy Manifest"
+      if: github.event_name == 'push' && env.SSH_PASSWORD != null && env.MANIFEST_DIRTY != null && env.MANIFEST_DIRTY != '0'
+      run: bash tools/deploy.sh manifest.json
+      env:
+        DESTDIR: public_html/downloads/
+        DEPLOY_ONLY: 1
+        SSH_HOST: downloads-hostgator.mixxx.org
+        SSH_KEY: packaging/certificates/downloads-hostgator.mixxx.org.key
+        SSH_PASSWORD: ${{ secrets.DOWNLOADS_HOSTGATOR_DOT_MIXXX_DOT_ORG_KEY_PASSWORD }}
+        SSH_USER: mixxx
+        UPLOAD_ID: ${{ github.run_id }}
+
+    - name: "Trigger Netlify build"
+      if: env.NETLIFY_BUILD_HOOK != null && env.MANIFEST_DIRTY != null && env.MANIFEST_DIRTY != '0'
+      run: curl -X POST -d {} ${{ env.NETLIFY_BUILD_HOOK }}
```

don't know if it matters: https://docs.netlify.com/configure-builds/build-hooks/
```suggestion
      run: curl -X POST -d '{}' ${{ env.NETLIFY_BUILD_HOOK }}
```